bin/local-env/launch-containers.sh

```diff
@@ -20,12 +20,12 @@ fi
 
 # Stop existing containers.
 status_message "Stopping Docker containers..."
-dc down --remove-orphans >/dev/null 2>&1
+dc down --remove-orphans
```

Comment:
This was what allowed us to see what the actual error was. It's probably best to avoid suppressing output (particularly STDERR) in such scripts as it is important for debugging.